src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((object, property) => {
+    const trimmed = property.trim();
```

Comment:
for what are you trimming here?

Reply:
It is for protection against empty property. For example 'color: white;      ; background-color: black;' would be split for 'color: white', '      ', background-color: black' and then the second string would be trimmed to '' and later compared to '' in if statement which would result returning unmodified object.